n3fit/src/n3fit/model_gen.py

```diff
@@ -170,6 +171,10 @@ def observable_generator(
         boundary_condition: dict
             dictionary containing the instance of the a PDF set to be used as a
             Boundary Condition.
+        mask_array: np.ndarray
```

Comment:
```suggestion
        training_mask_array: np.ndarray
```

I think this would help a lot with readability